ee/codegen/src/types/vellum.ts

```diff
@@ -898,10 +898,15 @@ type WorkflowSandboxInput =
   | ImageInputRequest
   | DocumentInputRequest;
 export type WorkflowSandboxInputs = WorkflowSandboxInput[];
+export interface WorkflowSandboxDatasetRowMockOutput {
+  output_id: string;
+  value: WorkflowValueDescriptor;
+}
+
 export interface WorkflowSandboxDatasetRowMock {
   node_id: string;
   when_condition?: WorkflowValueDescriptor;
-  then_outputs: Record<string, unknown>;
+  then_outputs: Record<string, unknown> | WorkflowSandboxDatasetRowMockOutput[];
```

Comment:
[blocking] this should never be the case from django... would love to see where this is coming from